config/api.yml

```diff
@@ -1590,3 +1590,16 @@
       :delete:
       - :name: delete
         :identifier: zone_delete
+  :container_deployments:
+    :description: Container Provider Deployment
+    :identifier: container_deployment
+    :klass: ContainerDeployment
+    :methods: *70174834085860
+    :options:
+    - :collection
+    :collection_actions:
+      :post:
+      - :name: start
+        :identifier: start
+      - :name: collect_data
+        :identifier: collect_data
```

Comment:
should also support resource_actions, to be able to do POST /api/container_deployments/:id/   action: "start" and such.